src/app/(nextjs_migration)/(authenticated)/user/layout.tsx

```diff
@@ -33,21 +33,11 @@ const MainLayout = async (props: Props) => {
     return <SignInButton autoSignIn />;
   }
 
-  const accountId = session?.user?.subscriber?.fxa_uid;
-
-  let userId = "";
-  if (accountId && typeof accountId === "string") {
-    // If the user is logged in, use a UUID based on the user's subscriber ID.
-    // TODO determine if we can collect the FxA UID directly https://mozilla-hub.atlassian.net/browse/MNTOR-2180
-    if (process.env.NIMBUS_UUID_NAMESPACE) {
-      userId = uuidv5(accountId, process.env.NIMBUS_UUID_NAMESPACE);
-    } else {
-      logger.error("NIMBUS_UUID_NAMESPACE env var not set");
-    }
-  }
+  // FIXME we need permission to collect this.
```

Comment:
Q: What does this mean?
Should we block merging of PR until we have said permission?
Or just not deploy this to production until permission is granted?

Reply:
I think data review should block merging, and if they say "no" to this part then this part of the instrumentation should be reverted.

Reply:
Data review is complete, I'll remove the FIXME.